pennylane/gradients/__init__.py

```diff
@@ -50,6 +50,20 @@
     hadamard_grad
     stoch_pulse_grad
     pulse_odegen
+    classical_jacobian

```

Comment:
Should this be in 'Utility functions' instead?

Reply:
Yes, it is true that it is more sense for it to be there. Because one can simply use any framework directly to get the parameters pre processing gradients. I am moving it.